README.md

```diff
@@ -23,11 +37,7 @@ Then visit: [http://localhost:8080/](http://localhost:8080/)
 ## Testing
 
 ### Acceptance Tests
-With the server running in development mode (`yarn run start:dev`), start the acceptance tests:
-
-```bash
-$ yarn run test:acceptance
-```
+From October 2024 release, acceptance tests are automated and are now running by QAT team in their environment. 
```

Comment:
Technically they're not actually automated because you are meant to run them as part of CI/CD.  It was automated before.  This requires someone to press a button which is against the normal/best practices.  I'm not sure you need to have the date

```suggestion
Acceptance tests are run by the QAT team in a different environment. 
```

Reply:
update as suggested